### PR TITLE
Update ATF with LDoc documentation

### PR DESCRIPTION
**Pull request by aderiabin**

### CODE REVIEW

File: modules/atf_logger.lua [hunk 1]
```diff
@@ -14,20 +32,28 @@ local Logger =
...
   }
 }
 
 Logger.mobile_log_format = "%s(%s) [version: %s, frameType: %s, encryption: %s, serviceType: %s, frameInfo: %s, messageId: %s] : %s \n"
 Logger.hmi_log_format = "%s[%s] : %s \n"
 
-function Logger.formated_time(withoutDate)
-  if withoutDate == true then
+--- Create string representation of current time in set format
+-- @tparam ?boolean withoutDate Set date format
```
Comment (VVeremjova) [inline modules/atf_logger.lua]: @aderiabin Is `?` special symbol here?

Comment (aderiabin) [inline modules/atf_logger.lua]: @VVeremjova Yes it is used for optional parameters. 
In other words:
 `--@tparam ?boolean withoutDate` and `@tparam boolean|nil withoutDate` the same.

File: modules/services/heartbeat_monitor.lua [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
 
-local module = {}
+local HbMonitor = {}
 local mt = { __index = { } }
 
+--- Type which represent Heartbeat monitor. It responsible for all heartbeat emulation activities.
+-- @type HeartBeatMonitor
+
+--- Prepare HeartBeatMonitor for perform heartbeat
 function mt.__index:PreconditionForStartHeartbeat()
```
Comment (VVeremjova) [inline modules/services/heartbeat_monitor.lua]: @aderiabin File was updated, there is no PreconditionForStartHeartbeat

File: modules/expectations/session_expectations.lua [hunk 3]
```diff
@@ -6,13 +18,16 @@ local Event = events.Event
 local SUCCESS = expectations.SUCCESS
```
Comment (VVeremjova) [inline modules/expectations/session_expectations.lua]: @aderiabin If you do a little refactoring, then remove these 2 variables as redundant

File: modules/SDL.lua [hunk 4]
```diff
@@ -55,9 +80,11 @@ function SDL:StartSDL(pathToSDL, smartDeviceLinkCore, ExitOnCrash)
   end
   xmlReporter.AddMessage("StartSDL", {["message"] = msg})
   return result, msg
-
 end
 
+--- Stop SDL from ATF
+-- @treturn nil The main result. Всегда nil.
```
Comment (VVeremjova) [inline modules/SDL.lua]: @aderiabin Russian

File: modules/hmi_connection.lua [hunk 5]
```diff
@@ -75,30 +110,40 @@ function module.mt.__index:SendError(id, methodName, code, errorMessage)
...
 
-function module.mt.__index:OnDisconnected(func)
+--- Set handler for OnDisconnected
+-- @tparam function func Handler function
+function HmiConnection.mt.__index:OnDisconnected(func)
   self.connection:OnDisconnected(function() func(self) end)
 end
 
-function module.mt.__index:Close()
+--- Close connection
```
Comment (VVeremjova) [inline modules/hmi_connection.lua]: @aderiabin Please mention that Close() only proxies calls to the next level

File: modules/mobile_session.lua [hunk 6]
```diff
@@ -142,21 +170,23 @@ function module.MobileSession(test, connection, regAppParams)
     return  res.correlationId
   end
 
-  -- heartbeat parameters
```
Comment (VVeremjova) [inline modules/mobile_session.lua]: @aderiabin Why did you remove description?

Comment (aderiabin) [inline modules/mobile_session.lua]: @VVeremjova Were added descriptions for each parameter separately.
